hook.py

```diff
@@ -2,15 +2,21 @@
 
 from app.utility.base_world import BaseWorld
 from plugins.atomic.app.atomic_svc import AtomicService
+from plugins.atomic.app.atomic_gui import AtomicGUI
 
 name = 'Atomic'
 description = 'The collection of abilities in the Red Canary Atomic test project'
-address = None
+address = '/plugin/atomic/gui'
 access = BaseWorld.Access.RED
 data_dir = os.path.join('plugins', 'atomic', 'data')
 
 
 async def enable(services):
+    atomic_gui = AtomicGUI(services, name, description)
+    app = services.get('app_svc').application
+    app.router.add_static('/atomic', 'plugins/atomic/static/', append_version=True)
```

Comment:
I think this is throwing an error since it doesn't exist. I think this line needs to be removed or you might have some unpushed `static` directory.

```
2021-05-05 17:01:42 - ERROR (c_plugin.py:70 enable) Error enabling plugin=atomic, No directory exists at '/Users/bworrell/code/caldera/plugins/atomic/static'
```

Reply:
> I think this is throwing an error since it doesn't exist. I think this line needs to be removed or you might have some unpushed `static` directory.
> 
> ```
> 2021-05-05 17:01:42 - ERROR (c_plugin.py:70 enable) Error enabling plugin=atomic, No directory exists at '/Users/bworrell/code/caldera/plugins/atomic/static'
> ```

You're right--thanks for catching that. There's no assets needed so I removed this line from the hook.py